src/app.ts

```diff
@@ -1,86 +1,51 @@
 import fastify from 'fastify'
```

Comment:
can you split this file into `api.ts` with startServer and `main.ts` that has the main function